server/instrumentation-backend/src/main/java/sh/calaba/instrumentationbackend/actions/location/FakeGPSLocation.java

```diff
@@ -142,14 +142,8 @@ private void setLocation(LocationManager locationManager, String locationProvide
             location.setLongitude(longitude);
             location.setAccuracy(1);
             location.setTime(System.currentTimeMillis());
-
-            try {
-                Method makeComplete = Location.class.getMethod("makeComplete");
-                if (makeComplete != null) {
-                    makeComplete.invoke(location);
-                }
-            } catch (Exception e) {
-                //Method only available in Jelly Bean
+            if Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN {
```

Comment:
You forgot brackets here:
if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN) {